communications/slack.md

```diff
@@ -0,0 +1,137 @@
+</p>
+
+- Slack offers one simple toggle to enter the "Do Not Disturb" mode. Use it to minimize distractions, so you can find what you need to focus on, faster.
+<p align="center">
+ <img width="500" alt="do not disturb" src=https://user-images.githubusercontent.com/53827690/89809928-46ab2a80-db0a-11ea-8e05-324750c2c748.png>
+</p>
+
+Right under your name, on the left side of slack, look for `Mentions & Reactions` to easily locate who mentioned you (tagged your username `@user` in a conversation) and who added a reaction to your message in a channel. A similar approach would be to trigger the command `/pings @user` from any chat to see how many times that person has been mentioned
+<p align="center">
+ <img width="500" alt="do not disturb" src=https://user-images.githubusercontent.com/53827690/89830184-cc8a9e00-db29-11ea-80ee-3d96732763c1.png>
+</p>
+
+
+
+### Threads
+
+Asynchronous communication can become chaotic so it should be every participant's (at least the initiator's) obligation to organize and summarize afterward and write it in well-formatted documentation as a part of our internal knowledge base. 
+
+- Currently, there's a trend to use `?!` to initiate a thread. It's meant to flag to all users and whoever replies to that message that a thread should be initiated. Therefore the subject gets it's own scope and context, it would not constantly trigger notifications to channel members, and avoid running the risk of someone else posting another topic amidst the first discussion. 
+
+- The number of conversations exchanged in a thread can be seen as an indicator of "significance" (or lower count often represents a "smooth" operation/confirmation without much discussion needed) and would facilitate other users to catch up on work after a day or a week off.
+
+- The `?!` also helps for future automations that can locate threads and return a summarized message per channel. Try it by using the commnand `/threads` in any channel. To have a better result, the message that initiates the thread needs to be build without any spaces: `?!release-atom`
+
+<p align="center">
+    <img width="500" alt="thread" src=https://user-images.githubusercontent.com/53827690/89817943-f2f30e00-db16-11ea-9d7c-60df273d3168.png>
+</p>
+
+- At a bare minimum, copy the transcript of relevant discussions into a place for future reference or refinement to be retained in the knowledge base. One option is to build a `?!Summary` of the discussion inside that thread and then click the red tag to save that message for your future reference.
+
+<p align="center">
+    <img width="500" alt="summary" src=https://user-images.githubusercontent.com/53827690/89812190-bc64c580-db0d-11ea-8d8e-6938f14e622e.png>
+</p>
+
+- Feel free to subscribe to any thread in case you wanna check the outcome of the conversation or if at any point you want to receive notifications about it. If you started the thread or have been tagged, Slack will subscribe you to it automatically.
```

Comment:
a user is auto-subscribed when:
1. the user started the thread
2. the user has been tagged
3. the user replied the thread

so probably something like

> If you started the thread, replied the thread, or have been tagged in the thread, ...